### PR TITLE
RSARTE-176125: Update tcjs files of lib-http-server

### PR DESCRIPTION
**Pull request by rajneshkumarmalik**
Updated tcjs files of lib-http-server

### CODE REVIEW

File: httpServerLib.tcjs [hunk 1]
```diff
@@ -15,13 +15,13 @@ tc.inclusionPaths = [
...
-tc.targetConfiguration = topTC.targetConfiguration || 'WinT.x64-VisualC++-15.0';
-tc.targetServicesLibrary = topTC.targetServicesLibrary || 'D:\\git\\rsarte-target-rts\\rsa_rt\\C++\\TargetRTS';
+tc.targetConfiguration = topTC.targetConfiguration || 'WinT.x64-VisualC++-17.0';
+tc.targetServicesLibrary = topTC.targetServicesLibrary;
 
 tc.unitName = 'UnitName_HttpLib';
 tc.copyrightText = `
 /*******************************************************************************
- * (c) Copyright HCL Technologies Ltd. 2019.  MIT Licensed!
+ * (c) Copyright HCL Technologies Ltd. 2024.  MIT Licensed!
```
Comment (mattias-mohlin) [inline httpServerLib.tcjs]: Copyright year should not be changed, but you can write 2019, 2024.

File: httpServerLib_static.tcjs [hunk 2]
```diff
@@ -22,13 +22,13 @@ tc.inclusionPaths = [
...
 
-tc.targetConfiguration = topTC.targetConfiguration || 'WinT.x64-VisualC++-15.0';
+tc.targetConfiguration = topTC.targetConfiguration || 'WinT.x64-VisualC++-17.0';
 tc.targetServicesLibrary = topTC.targetServicesLibrary || 'D:\\git\\rsarte-target-rts\\rsa_rt\\C++\\TargetRTS';
 
 tc.unitName = 'UnitName_HttpLib';
 tc.copyrightText = `
 /*******************************************************************************
- * (c) Copyright HCL Technologies Ltd. 2019.  MIT Licensed!
+ * (c) Copyright HCL Technologies Ltd. 2024.  MIT Licensed!
```
Comment (mattias-mohlin) [inline httpServerLib_static.tcjs]: see above

**Review by mattias-mohlin — APPROVED** [review]
ok, but please check comment about copyrights

File: httpServerLib_static.tcjs (hunk 2 above)
Comment (rajneshkumarmalik) [reply httpServerLib_static.tcjs]: corrected.

File: httpServerLib.tcjs (hunk 1 above)
Comment (rajneshkumarmalik) [reply httpServerLib.tcjs]: corrected

**Pull request merged**